api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -689,6 +694,9 @@ public static String getOpenmrsProperty(String property) {
 		props.add(new GlobalProperty(GP_MAIL_SMTP_STARTTLS_ENABLE, "false",
 		        "Set to true to enable TLS encryption, else set to false"));
 		
+		props.add(new GlobalProperty(GP_HOST_URL, "",
+		        "The activationKey is the parameter which will be generated and substituted. always make sure to provide a parameter name activationkey in the link you provide here "));
```

Comment:
A full stop can't be followed by a lower case letter

Reply:
This is actually wrong, don't dictate that the URL should have a parameter because that's a implementation detail

Reply:
You should provide a proper description of the GP, this doesn't feel like one